ddspipe_core/src/cpp/efficiency/payload/CopyPayloadPool.cpp

```diff
@@ -28,22 +28,19 @@ using namespace eprosima::ddspipe::core::types;
 
```

Comment:
Maybe you can add this to reduce verbosity:
```suggestion
using namespace eprosima::fastdds::rtps;
```